solutions/standard/variables.tf

```diff
@@ -82,3 +90,41 @@ variable "cbr_rules" {
   default     = []
   # Validation happens in the rule module
 }
+
+########################################################################################################################
+# KMS variables
+########################################################################################################################
+
+variable "existing_kms_instance_crn" {
+  type        = string
+  default     = "crn:v1:bluemix:public:kms:us-south:a/abac0df06b644a9cabc6e44f55b3880e:d4559998-17ab-4ab8-ba9e-a0c9d2c4ef23::"
+  description = "The CRN of the key management service (KMS). If you are not using an existing KMS instance, you must specify this CRN. To support cross account functionality you must also provide a value for `ibmcloud_kms_api_key`."
+}
+
+variable "existing_kms_key_crn" {
+  type        = string
+  default     = null
+  description = "Optional. The CRN of an existing root key in key management service (KMS) to encrypt the payload data in event streams. To create a key ring and key, pass a value for the `existing_kms_instance_crn` input variable. To support cross account functionality you must also provide a value for `ibmcloud_kms_api_key`."
+}
+
+variable "kms_endpoint_type" {
+  type        = string
+  description = "The type of endpoint to use to communicate with the key management service (KMS). Specify one of the following values for the endpoint type: `public` or `private` (default). To support cross account functionality you must also provide a value for `ibmcloud_kms_api_key`."
+  default     = "private"
+  validation {
+    condition     = can(regex("public|private", var.kms_endpoint_type))
+    error_message = "Valid values for the `kms_endpoint_type_value` are `public` or `private`. "
+  }
+}
+
+variable "es_key_ring_name" {
+  type        = string
+  default     = "es-key-ring"
+  description = "The name of the key ring to create for the Event Streams instance. If an existing key is used, this variable is not required. If the prefix input variable is passed, the name of the key ring is prefixed to the value in the `prefix-value` format."
```

Comment:
can we replace `If an existing key is used, this variable is not required. If the prefix input variable is passed, the name of the key ring is prefixed to the value in the `prefix-value` format.`  with `Not used if supplying an existing key. If a `prefix` input variable is specified, it is added to this name in the `<prefix>-value` format.` to be in sync with descriptions in another modules. for example https://github.com/terraform-ibm-modules/terraform-ibm-event-notifications/blob/main/solutions/standard/variables.tf#L128

Reply:
Made the required changes, very similar to - https://github.com/terraform-ibm-modules/terraform-ibm-observability-da/blob/main/solutions/instances/variables.tf#L335
This description is reviewed and approved by editors.